### ISSUE TITLE
Some hosts can't connect to each other

### ISSUE THREAD

**Issue by golfvert** [zerotier/ZeroTierOne#1542]
I am using zerotier for accessing personal servers for family related home automation installations.
Up to recently it was working flawlessly. All of a sudden, I can't connect from my mac (zerotier 1.8.4) or a VM (linux 1.8.4) to another server Linux 1.6.5.
However, my android phone on the same wifi network as the mac or VM Linux can connect to the remote one.
I then tried using the android phone as hotspot for the mac, no luck either.
I don't have a firewall on the mac or on the VM. My home wifi network doesn't prevent outgoing connections (android works anyway). 
The remote site has no filtering either.
I have unload/load on the mac. Downgraded to 1.6.5. Looking at zerotier-cli output nothing really pops out.
I have tried to change the private IP addressing scheme.
Basically, I have tried "everything" that I can think of without improvement.
Any suggestion on what I can do?

**Comment by golfvert** [comment]
Question asked on discuss... Sorry for the misuse.
